### DISCUSSION TITLE
How to get m3u8 link from youtube live stream?

### DISCUSSION THREAD

**Discussion by Manipulativemachan** [iptv-org/community#287] [Q&A] [answered]
I have been trying to add few news channels to the source. But the manifested youtube links are not working in tvimate pro. But few are working. Is there any way to get m3u8 link for youtube live stream?

**Comment by PD-TheKrazy0ne** [comment, marked as answer]
here is an example, it will play on most media players as well as a kodi playlist.
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/NatGeo/live

just substitute the youtube link with whatever you want to view

**Comment by Manipulativemachan** [comment]
Not working now. Getting error
